inference-engine/src/mkldnn_plugin/nodes/mkldnn_tile_node.cpp

```diff
@@ -56,7 +50,7 @@ void MKLDNNTileNode::initSupportedPrimitiveDescriptors() {
     config.inConfs[0].desc = MKLDNNMemoryDesc(getParentEdgeAt(0)->getDims(), inputDataType, fmt);
     config.outConfs[0].inPlace = -1;
     config.outConfs[0].constant = false;
-    config.outConfs[0].desc = MKLDNNMemoryDesc(getChildEdgeAt(0)->getDims(), outputDataType, fmt);
+    config.outConfs[0].desc = MKLDNNMemoryDesc(getChildEdgeAt(0)->getDims(), inputDataType, fmt);
```

Comment:
In general current implementation will not be valid for BIN data type (and others like INT4, etc). So my proposal is to check precision size (it should be equal 1,2,4 or 8). and throw exception otherwise.

Reply:
Done